lib/server.js

```diff
@@ -6,8 +6,25 @@ const path = require('path');
 const fs = require('fs');
 const spawn = require('child_process').spawn;
 const ffmpeg = require('fluent-ffmpeg');
+var https = require('https');
 
-module.exports = (directory, format, width, height, framerate, horizontalFlip, verticalFlip, compressionLevel, time, listSize, storageSize, port) => {
+module.exports = (
+  directory,
+  format,
+  width,
+  height,
+  framerate,
+  horizontalFlip,
+  verticalFlip,
+  compressionLevel,
+  time,
+  listSize,
+  storageSize,
+  port,
+  secure,
+  certificate,
```

Comment:
Would prefer that the `certificate` and `key` variable names indicate that they are paths rather than the entity they represent, e.g. `certificatePath` and `keyPath`.